daemons/gptp/gptp_cfg.ini

```diff
@@ -6,6 +6,11 @@
 # The lower the number, the higher the priority for the BMCA.
 priority1 = 248
 
+# Option to specify system clock type
```

Comment:
Can we add some more details here. Specifically, we should list the currently supported options, the default option and what the options mean. The text that is part of the pull request has great information in it, but I think that 6 months from now readers will prefer to read an updated comment here.

Reply:
Yes, I found comments in the example ini files very helpful when trying to actually use the avtp_pipeline code, so it'd be a nice help for users.